website/src/parsers/js/acorn.js

```diff
@@ -24,6 +24,9 @@ export default {
   },
 
   parse(parsers, code, options={}) {
+    if (Object.keys(options).length === 0) {
+      options = this.getDefaultOptions();
+    }
```

Comment:
Same question as in the other MR: Did you add this because you ran into issues?

Reply:
After pressing the "reset" button, `{}` was passed. So I added this branch. The acorn parser crashes with an error even before I make my changes.

![image](https://user-images.githubusercontent.com/16508807/121178346-5ebe1880-c899-11eb-8be4-331303e7468e.png)

![image](https://user-images.githubusercontent.com/16508807/121178323-54038380-c899-11eb-858b-e1e752e01279.png)


I referred to another parser.
https://github.com/fkling/astexplorer/blob/17d7bf5e628a2d3e3ca4f7cec4a14d531811c881/website/src/parsers/regexp/regexpp.js#L26